openstates/importers/base.py

```diff
@@ -180,7 +181,24 @@ def resolve_bill(self, bill_id: str, *, date: str) -> typing.Optional[_ID]:
         if len(ids) == 1:
             return ids.pop()
         elif len(ids) == 0:
-            self.error(f"could not resolve bill id {bill_id} {date}, no matches")
+            self.warning(f"could not resolve bill id {bill_id} {date}, no matches so will retry")
+            date = datetime.fromisoformat(date)
+            new_date = date + timedelta(days=-4)
```

Comment:
If the match failure is an off-by-one issue, why subtract four days instead of just one?

It seems like the biggest risk around this is overmatching on something like a special session that starts the day after a normal session - would a 4 day adjustment make that worse?

Reply:
That's a good point, I don't remember why Alex & I settled on 4 days. I brought up that we could tweak the date by one & if that didn't work then try a week, so I think 4 days was middle ground